### PR TITLE
* Add null check to Physic and Sprite observables for performance viewer

### PR DESCRIPTION
**Pull request by carolhmj**

### CODE REVIEW

**Review by sebavan — APPROVED** [review]

File: dist/what's new.md [hunk 1]
```diff
@@ -385,6 +385,7 @@
 - Fixed inTangent in animationGroup ([dad72](https://github.com/dad72))
 - Fixed bug in `QuadraticErrorSimplification` not correctly optimizing mesh. ([aWeirdo](https://github.com/aWeirdo))
 - Fixed bug in `ArcRotateCamera` where setting the position would recalculate the alpha value to a value outside the current limits. ([nilss0n](https://github.com/nilss0n))
+- Fix Perf Viewer crashing if Physics or Sprite strategies are used in scenes that don't include these features. ([carolhmj](https://github.com/carolhmj))
```
Comment (sebavan) [inline dist/what's new.md:388]: This is the wrong what s new :-) and we do not need update as this has not been shipped yet, could you revert this change ?

**Review by sebavan — CHANGES_REQUESTED** [review]
Just the whats new change :-)

**Review by RaananW — APPROVED** [review]
What @sebavan said 😊

File: dist/what's new.md (hunk 1 above)
Comment (carolhmj) [reply dist/what's new.md:388]: Dang, I got it wrong again, I swear I'll learn one day 🤣

Comment (carolhmj) [reply dist/what's new.md]: Reverted!
